src/network/simulated/core.rs

```diff
@@ -80,7 +81,10 @@ impl SimulatedNetworkCore {
                     let msg = guard.pop().unwrap();
                     let n_guard = n.read().await;
                     let channel = n_guard.get(&msg.to).unwrap();
-                    channel.send(msg).await.unwrap();
+                    if let Err(_e) = channel.send(msg).await {
+                        println!("sending failed. Ignoring");
```

Comment:
using print so that it shows up in tests without having to setup logging which is tricky to do.